compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1344,8 +1344,19 @@ impl Target {
             });
         }
 
-        dl.c_enum_min_size = match Integer::from_size(Size::from_bits(self.c_enum_min_bits)) {
-            Ok(bits) => bits,
+        dl.c_enum_min_size = match self
```

Comment:
Can this `match` be replaced with `.map_err(|err| TargetDataLayoutErrors::InvalidBitsSize { err })?`?

Reply:
Apparently yes. I had tried that several times but it didn't work before. I guess it decided working now is fine.